### DISCUSSION TITLE
Hi everyone!

### DISCUSSION THREAD

**Discussion by F-U-Njoku** [OpenMined/courses#37] [Introduce Yourself]
Hi all,
I am Uche, currently doing my masters' in big data management and analytics.
I look forward to learning with you! Cheers.

**Comment by yemikifouly** [comment]
Hi Uche! 👋 Nice meeting you! I am Yemi. Welcome to the discussion board! 🙂 I wonder what made you interested in the Private AI Series. Does your interest come from you studying big data or do you have an adjacent interest in privacy?

**Reply by F-U-Njoku** [reply]
Thank you, Yemi :-). I actually was introduced to privacy AI through the Udacity scholarship and that sparked my interest. It wasn't from my studies; we don't learn the privacy side of big data management in my studies.

**Reply by yemikifouly** [reply]
I was in that scholarship as well. 😀 Small world!

**Comment by iamtrask** [comment]
Welcome Uche! Congrats on entering your masters program! Hope you enjoy the course!

**Reply by F-U-Njoku** [reply]
Thank you Andrew :)
